frameworks/web-ui/react/src/App.tsx

```diff
@@ -1,33 +1,59 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import { Button } from './Button'
+
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [equation, setEquation] = useState<string>("0");
+
+  const addToEquation = (token: string) => {
+    setEquation((equation) => {
+      if (equation == "0") {
+        return token;
+      }
+      return equation + token
+    });
+  }
+
+  const clear = () => {
+    setEquation((equation) => "0");
```

Comment:
Why are you setting the state of `equation` with a function that returns the string "0"?
This can just be `setEquation("0")`, can it not?